spec/wasabi/document/escaped_endpoint_spec.rb

```diff
@@ -0,0 +1,15 @@
+
+require "spec_helper"
+
+describe Wasabi::Document do
+  context "with: escaped_endpoint.wsdl" do
+
+    subject { Wasabi::Document.new fixture(:escaped_endpoint).read }
```

Comment:
```suggestion
    subject(:document) { Wasabi::Document.new fixture(:escaped_endpoint).read }
```

By naming `subject`, we can keep them easier to spot over time. I enjoy reading "less-technical" RSpec tests.